src/map/common/lapacksrc/netlib/install.list

```diff
@@ -1,8 +1,4 @@
-ieeeck.f
-ilaenv.f
-iparmq.f
 lsamen.f
-xerbla.f
```

Comment:
This file blocks the install of all these files. Meaning they now get added into libFlame when built